zinit-install.zsh

```diff
@@ -1678,7 +1678,7 @@ ziextract() {
         ((#i)*.rar)
             →zinit-extract() { →zinit-check unrar "$file" || return 1; command unrar x "$file"; }
             ;;
-        ((#i)*.tar.bz2|(#i)*.tbz2)
+        ((#i)*.tar.bz2|(#i)*.tbz|(#i)*.tbz2)
```

Comment:
This can be reduced to avoid needing to duplicate the `tbz` but maybe not desired for readability.

LGTM otherwise, nice change.

Reply:
I must admit I'm drawing a blank here 😕 I've tried `tbz?`, `tbz?(2)` and even `tbz2?` (regex-style), but none of those variations worked. Do you mind helping me out here? Or do we want to keep it as it is?

Reply:
Lgtm like it is. Readability is king in my eyes. We don't want zinit to get more cryptic ;)

Reply:
Hehe, alright :)